nuxeo-mongodb-apb/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+apb_name: nuxeo-mongodb-apb
+
+default_debug: True
+bootstrap_debug_enabled: False
+
+# Defaults for values we could expect from a broker
+_apb_plan_id: secure-replicaset
+_apb_service_class_id: 0
+_apb_service_instance_id: 0
+
+state_map:
+  provision: present
+  deprovision: absent
+state: "{{ state_map[apb_action] }}"
+
+cluster: kubernetes
+
+name: "{{ apb_name }}-{{ _apb_service_instance_id }}"
```

Comment:
I would rather use `nuxeo-mongodb-{{ _apb_service_instance_id }}` as the name: unless there is a convention on that, I don't see the point of having that `-apb` suffix at the end of the name. 
There are already labels on each resource to know it comes from an APB but the resource itself is not an APB.

Reply:
This is patterned after the defaults/main.yml in the "Modernized Hello World APB" available here: https://github.com/ansibleplaybookbundle/hello-world-apb/blob/master/defaults/main.yml. 

I find it interesting that the following namespace definition namespace: "{{ lookup('env','NAMESPACE') | default('hello-world', true) }}", uses "hello-world" without the apb suffix whereas the name definition includes the apb suffix. I agree with you that it makes more sense for these two values to be consistent and that the apb suffix offers no value. I will revise accordingly.